.github/workflows/canary-release.yml

```diff
@@ -20,10 +20,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Setup Node.js 18
-        uses: actions/setup-node@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: lts/*
```

Comment:
I updated this node version based on syntax I saw available [here](https://github.com/actions/setup-node?tab=readme-ov-file#supported-version-syntax). AFAIK this is the way to say "always use an LTS version" (so we're on something stable), but should also give us the latest (cached) version of LTS so we don't have to come in here and update the node version regularly.

Reply:
The one problem with this is it could cause our jobs to start breaking if a new major LTS version comes out and our setup doesn't work with it. Probably not the biggest deal, but should we pin to 20 to avoid that?

Reply:
I'm not too worried about a new LTS version breaking our jobs, because we'd notice it as part of our work (these CI jobs wouldn't run without a PR being created or changes being pushed up). 

I had noticed that we've tweaked the CI files multiple times in the past 2 years, mainly to bump the node version.